### ISSUE TITLE
DLC can't be retrieved

### ISSUE THREAD

**Issue by bookmountain** [aizenar/PS4_db_Rebuilder_EXT#12]
I know it's an existed issue since this tool was released, may you fix this?
there are so many DLC in my hard-drive, is there anyway to restore them from db?
Thank you very much.

**Comment by Pharaoh2k** [comment]
This specific script does rebuild the database and add the DLCs too.
Are you referring to DLCs on the internal drive or the external/extended storage?

**Comment by bookmountain** [comment]
External drive's DLCs are restored, but not in the internal drive. Firmware 9.00 model 1207

**Comment by Pharaoh2k** [comment]
Yes, it is not designed to retrieve DLCs. You will need to reinstall only the DLCs.

**Comment by Pharaoh2k** [comment]
Use Itemzflow, which works better than any script, including DLCSs:
https://github.com/LightningMods/Itemzflow

**Comment by bookmountain** [comment]
Thanks for the advice, appreciated
